frontend/coprs_frontend/coprs/views/webhooks_ns/webhooks_general.py

```diff
@@ -79,6 +80,21 @@ def decorated_function(copr, **kwargs):
 
     return decorated_function
 
+def addHookHistoryRecord(webhook_uuid, user_agent, builds_initiated_via_hook):
+    '''This methods adds the intercepted webhook to webhook_history db.'''
+    if(webhook_uuid is not None):
+        for build in builds_initiated_via_hook:
+            webhookRecord = models.WebhookHistory(webhook_uuid=webhook_uuid,
+                                                  user_agent=user_agent,
+                                                  build_id=build)     
+            try:
+                db.session.add(webhookRecord)
+            except:
```

Comment:
You typically don't want a broad `except` statement like this or `except Exception`. When you make any error in the `try` block, even something like using a wrong variable name, it will be handled by the `except` block. It makes it really hard to debug. I guess we can catch something SQLAlchemy-specific here.

Reply:
Removed the `except:` clause as a whole. Instead added  default values to the parameters of function `def add_webhook_history_record(webhook_uuid, user_agent='Not Set', builds_initiated_via_hook=None):` and a validation check within method which made more sense than throw an exception.